.github/workflows/build-docker-image.yml

```
@@ -68,6 +68,7 @@ jobs:
       - name: Remove leading v from the version
         run: |
           echo "VERSION=$(echo ${{ github.ref_name }} | sed 's/^v//')" >> $GITHUB_ENV
```

Useless use of `echo`.

```suggestion
          echo "VERSION='$(sed 's/^v//' <<< ${{github.ref_name}})'" >> $GITHUB_ENV
```

---

This is the pattern used in the GitHub workflow documentation. I don't think it is any less easy to read or understand than binding stdin. 

I find the pipe clearer to understand the flow of data than the way you wrote it.